packages/cli/src/linter/base-linter.ts

```diff
@@ -0,0 +1,55 @@
+import { Spectral, Document, ISpectralDiagnostic } from '@stoplight/spectral-core'
+import { Json as JsonParser, JsonParserResult } from '@stoplight/spectral-parsers'
+import { Logger } from '../logger'
+
+export abstract class BaseLinter {
```

Comment:
**chore:** no need to have this abstract class since there is currently a single class extending it

Reply:
This was done so that we can easily extend it to an InterfaceLinter when needed, but I agree, it's premature